app/controllers/submissions_controller.rb

```diff
@@ -153,6 +153,16 @@ def populate_file_manager
   def manually_collect_and_begin_grading
     assignment = Assignment.find_by(id: params[:assignment_id])
     @grouping = assignment.groupings.find(params[:grouping_id])
+    released = Grouping.joins(current_submission_used: :results)
+                       .where('results.released_to_students': true)
+                       .where(id: @grouping)
```

Comment:
You can make this more idiomatic Rails by using associations: `@grouping.current_submission_used.results.where(...)`